BackofficeBundle/Resources/public/templateSet/default/column_left.html

```diff
@@ -0,0 +1,40 @@
+<div class="row">
+    <div class="col-lg-12">
+        <div class="area header disabled">
+            <span>Header</span>
```

Comment:
We should find a way to use translation here : if the BO is turned in french, this won't be turned too